app/components/Speakers/EditSpeakerForm.jsx

```diff
@@ -19,6 +19,14 @@ const validate = ({ full_name, title }) => {
 @withNamespaces('videoDebate')
 export default class EditSpeakerForm extends React.PureComponent {
   render() {
+    const maxTitleLen = SPEAKER_TITLE_LENGTH[1]
+    const titleLabel =
```

Comment:
This is not easy to read, can we use string interpolation here instead? Something like:

```es6
const charactersStr = t('main:misc.character', { count: maxTitleLen })
const titleLabel = `${t('speaker.title')} (\u2264 ${maxTitleLen} ${charactersStr})
```

Reply:
This is much better, thanks!